web/src/pages/guidance/published.page.tsx

```diff
@@ -74,98 +71,135 @@ export default function Published({
 
 			<GuidanceListNav />
 
-			<p>
-				Showing {products.length} products on page {currentPage} of {totalPages}{" "}
-				({totalProducts} products total)
-			</p>
-
 			<Grid gutter="loose">
 				<GridItem cols={12} md={4} lg={3}>
-					<h2>Product types</h2>
-					<ul className="list list--unstyled">
-						{productTypes.map(({ Name }) => (
-							<li key={Name}>{Name}</li>
-						))}
-					</ul>
-					<h2>Areas of interest</h2>
-					<ul className="list list--unstyled">
-						{areasOfInterest.map(({ Name }) => (
-							<li key={Name}>{Name}</li>
+					<FilterPanel heading="Filter">
+						<FilterByInput
+							name="q"
+							label="Filter by title"
+							buttonLabel="Filter by title"
+						/>
+						{results.navigators.map(({ shortName, displayName, modifiers }) => (
+							<FilterGroup
+								key={shortName}
+								heading={displayName}
+								selectedCount={
+									modifiers.filter((modifier) => modifier.active).length
+								}
+							>
+								{modifiers.map((modifier) => (
+									<FilterOption
+										key={modifier.displayName}
+										isSelected={modifier.active}
+										onChanged={() => {
+											/* what */
+										}}
+										value={modifier.displayName}
+									>
+										{modifier.displayName}
+									</FilterOption>
+									// <li key={modifier.displayName}>
+									// 	<Link href={modifier.toggleUrl.fullUrl}>
+									// 		<a>{modifier.displayName}</a>
+									// 	</Link>
+									// </li>
+								))}
+							</FilterGroup>
 						))}
-					</ul>
+					</FilterPanel>
 				</GridItem>
 				<GridItem cols={12} md={8} lg={9}>
-					<h2>Products</h2>
-					<ol className="list list--unstyled">
-						{products.map((product) => (
-							<ProductCard
-								key={product.Id}
-								product={product}
-								productTypeName={getProductTypeName(product)}
-							/>
-						))}
-					</ol>
+					<FilterSummary
+						id="filter-summary"
+						activeFilters={activeModifiers.map((modifier) => ({
+							label: modifier.displayName,
+							destination: modifier.toggleUrl.fullUrl,
+						}))}
+						sorting={[{ label: "Date" }, { label: "Title" }]}
+					>
+						Showing {results.firstResult} to {results.lastResult} of{" "}
+						{results.resultCount}
+					</FilterSummary>
+
+					<Table aria-describedby="filter-summary">
+						<thead>
+							<tr>
+								<th>Title</th>
+								<th>Reference Number</th>
+								<th>Published</th>
+								<th>Last updated</th>
+							</tr>
+						</thead>
+						<tbody>
+							{results.documents.map(
+								({
+									id,
+									title,
+									guidanceRef,
+									publicationDate,
+									lastUpdated,
+									sourceUrl,
+								}) => {
+									const strippedTitle = guidanceRef
+										? title.replace(`(${guidanceRef})`, "").trim()
+										: title;
+									return (
+										<tr key={id}>
+											<td>
+												<a href={sourceUrl}>{strippedTitle}</a>
```

Comment:
Using `pathAndQuery` rather than `sourceUrl` probably makes more sense so the URL is relative - then we can click the result and stay within the same environment, otherwise we'd always end up on live

Reply:
Also, eventually we'd turn these anchors into NextJS `Link` components for client side routing (once we migrate guidance products into next-web) so the path would _have_ to be relative